physics/mass_energy_equivalence.py

```diff
@@ -0,0 +1,77 @@
+"""
+Title:
+Finding the energy equivalence of mass and mass equivalence of energy
+by Einstein's equation.
+
+Description:
+Einstein's mass-energy equivalence is a pivotal concept in theoretical physics.
+It asserts that energy (E) and mass (m) are directly related by the speed of
+light in vacuum (c) squared, as described in the equation E = mc². This means that
+mass and energy are interchangeable; a mass increase corresponds to an energy increase,
+and vice versa. This principle has profound implications in nuclear reactions,
+explaining the release of immense energy from minuscule changes in atomic nuclei.
+
+Equations:
+E = mc² and m = E/c², where m is mass, E is Energy, c is speed of light in vacuum.
+
+Reference:
+https://en.wikipedia.org/wiki/Mass%E2%80%93energy_equivalence
+"""
+
+from scipy.constants import c  # speed of light in vacuum (299792458 m/s)
+
+
+def energy_from_mass(mass: float) -> float:
+    """
+    Calculates the Energy equivalence of the Mass using E = mc²
+    in SI units J from Mass in kg.
+
+    mass (float): Mass of body.
+
+    Usage example:
+    >>> energy_from_mass(124.56)
+    1.1194894506345802e+19
+    >>> energy_from_mass(320)
+    2.8760165719578165e+19
```

Comment:
```suggestion
    >>> energy_from_mass(124.56)
    1.11948945063458e+19
    >>> energy_from_mass(320)
    28760165719578164480
```
Fixing doctests (because for some reason the doctest output is now less precise than before)